app.css

```diff
@@ -458,9 +458,16 @@ button:hover {
      text-overflow: ellipsis;
 }
 .main__cutter-shortened-links p:nth-child(3) {
-    color: var(--cyan);
     margin: 0 20px 0 10%;
 }
+.main__cutter-shortened-links p:nth-child(3) a {
+    color: var(--cyan);
+    text-decoration: none;
+}
+.main__cutter-shortened-links p:nth-child(3) a:hover {
+    color: var(--cyan);
```

Comment:
Great job! 
Could you please take out `Line 468` since we only need the link underlined when the user hovers over it?